test/functional/cmdLineTests/criu/criu_nonPortable.xml

```diff
@@ -205,6 +205,22 @@
     <output type="failure" caseSensitive="yes" regex="no">User requested Java dump using</output>
   </test>
 
+  <test id="Create CRIU checkpoint image and restore once - testGetProcessRestoreStartTime">
+    <command>bash $SCRIPPATH$ $TEST_RESROOT$ $JAVA_COMMAND$ "$JVM_OPTIONS$ -Xtrace:print={j9vm.684-696,j9vm.699,j9vm.717-746} --add-exports java.base/openj9.internal.criu=ALL-UNNAMED" $MAINCLASS_TIMECHANGE$ testGetProcessRestoreStartTime 1 false false</command>
```

Comment:
Why 746 here, but 745 on line 193? I think both should be 745.

Reply:
I will set `-Xtrace:print=` to `{j9vm.684-696,j9vm.699,j9vm.717-746}` for both `testGetProcessRestoreStartTime` and `testGetProcessRestoreStartTime` as `j9vm.745` is `Trc_VM_criu_after_dump` and `j9vm.746` is `Trc_VM_criu_process_restore_start_after_dump`.

Reply:
I misread `J9TraceFormat.dat`: yes, both should be 746.